app/scripts/lib/uns-ipfs/resolver.js

```diff
@@ -0,0 +1,8 @@
+import Resolution from "@unstoppabledomains/resolution";
+
+export default async function resolveUnsToIpfsContentId(domainName) {
+    let result;
+    const resolution = new Resolution();
+    result = await resolution.ipfsHash(domainName);
```

Comment:
Nitpick: why not use `const` instead?

```suggestion
    const result = await resolution.ipfsHash(domainName);
```

Reply:
it's a good catch tho. there are definitely some lets in our code that should be a const